ddd-and-spring/index.adoc

```diff
@@ -114,26 +114,26 @@ Domain services implement functionality that cannot uniquely be assigned to an e
 
 As the names suggest, the differentiating line is drawn between the ways a developer gets hold of an instance of a class for that particular model element.
-A newable can be simply instantiated using the `new` operator, although that should even be limited to as little places as possible. The factory pattern can help here, too.
+A newable can be simply instantiated using the `new` operator, although even that should be limited to as few places as possible. The factory pattern can help here, too.
 Entities and value objects are newables.
 An injectable is usually a Spring component, which means that the latter controls its lifecycle, creates instances and destroys them. This allows the container to equip the service instance with technical services like transactions or security.
 Clients obtain instances by using dependency injection (hence the name injectable).
 Repositories and services are injectables.
 
-This distinction between these thow groups of classes naturally defines a preferred dependency direction from injectables to newables. Generally speaking
+This distinction between these two groups of classes naturally defines a preferred dependency direction from injectables to newables. Generally speaking
 
 - _Value object_ - JPA `@Embeddable` + corresponding `equals(…)` and `hashCode()` (Lombok's `@Value` helps here). Can depend on other value objects and entities.
 - _Entity_ - JPA `@Entity` + corresponding `equals(…)` and `hashCode()` implementations. Can depend on other entities and value objects.
 - _Repository_ - Spring component, usually a Spring Data repository interface. Can depend on entities and value objects, are centered around entities that are aggregate roots.
 - _Domain services_ - Usually a Spring component, a class annotated with `@Component` or a stereotype annotation. Can also be modeled as newables in some cases in case they don't require technical services to be applied (e.g. security, transactions).
 
-Not all classes of a Spring application can be assigned to any of these DDD categories. These other classes can usually be grouped into these more technical ones:
+Not all classes of a Spring application can be assigned to these DDD categories. These other classes can usually be grouped into the following, more technical ones:
 
 - _Application configuration_ - Classes to configure components.
-- _Technical adapters_ - Business logic implemented in a Spring application is usually exposed to clients through some remoting technology. In a web application these technologies are HTTP, HTML and JavaScript. With Spring MVC, controller classes serve the purpose of translating the concepts of the remoting technology (e.g. the notion of a request, request parameters, a payload, a response etc.) into the domain concepts and invoke services with value objects and entities.
```

Comment:
I am gonna keep "remoting" as that's the term used to describe that functionality, see [the Spring reference documentation](http://docs.spring.io/spring/docs/current/spring-framework-reference/html/remoting.html).